org.eclipse.jdt.ls.core/src/org/eclipse/jdt/ls/core/internal/cleanup/MemberAccessUsesThisCleanUp.java

```diff
@@ -28,8 +29,8 @@
 public class MemberAccessUsesThisCleanUp implements ISimpleCleanUp {
 
 	@Override
-	public String getIdentifier() {
-		return "qualifyMembers";
+	public Collection<String> getIdentifiers() {
+		return List.of("qualifyMembers");
```

Comment:
No interest in enabling this & qualifyStaticMembers ?

Reply:
Those cleanups don't map to a single JDT-UI cleanup id. JDT-UI has a single cleanup id for "style" which delegates to other configurations. The grain is different in JDT-LS to easily map those cleanups to a JDT-UI id